openy_calc/openy_calc.module

```diff
@@ -48,3 +71,74 @@ function openy_calc_preprocess_fieldset(&$variables) {
     $legend_span->setAttribute('tabindex', '0');
   }
 }
+
+/**
+ * Remove branch_hours from the related branch for "facility" node viewing.
+ */
+function openy_calc_preprocess_node__branch__calc_summary(&$variables) {
+  $node = $variables['node'];
+  if(!$node instanceof NodeInterface) {
+    return;
+  }
+
+  $variables['branch'] = [
+    'branch_name' => [
+      '#type' => 'inline_template',
+      '#template' => '<h4>{{ branch_name }}</h4>',
+      '#context' => [
+        'branch_name' => $node->label(),
+      ],
+    ],
+    'branch_phone' => [
+      '#type' => 'inline_template',
+      '#template' => '<a href="tel:{{ branch_phone }}"><i class="fa fa-phone" aria-hidden="true"></i><div>{{ branch_phone }}</div></a>',
+      '#context' => [
+        'branch_phone' => $node?->get('field_location_phone')->value,
```

Comment:
seems like we do expect $node could be NULL here which will make inline template on line 94 looks broken. We might need to have a condition, once $node is NULL - disaplay nothing or display placeholder @aleevas

Reply:
On line 80 we have condition $node, which it  instanceof NodeInterface so $node canot be NULL